### PR TITLE
New standard and csv outputs

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description
New CSV and standard output for comparisons, the different UUIDs have now their own column. The CSV output is quite useful to print graphs from results (It fits the format Gsheet expects to generate graphs):

```shell
$ touchstone_compare --database elasticsearch -url https://es.server:443   -u 04687258-a6c9-433c-ad7b-94f157b6f2bd 0de042ff-bfd9-4f33-98f6-8639edfa0d06  --config config/mb.json   --alias OpenShiftSDN OVN  
 --tolerancy-rules tolerancy-configs/mb.yaml                                                                       
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+----------+
| test_type | routes | conn_per_targetroute | keepalive |          metric          | result | deviation | OpenShiftSDN |   OVN    |
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+----------+
|   http    |  100   |          1           |     0     | avg(requests_per_second) |  Pass  |  -13.69%  |   59251.0    | 51141.5  |
|   http    |  100   |          1           |     1     | avg(requests_per_second) |  Pass  |  -6.37%   |   21802.5    | 20414.5  |
|   http    |  100   |          1           |    50     | avg(requests_per_second) |  Fail  |  -17.30%  |   56382.5    | 46628.5  |
|   http    |  100   |          40          |     0     | avg(requests_per_second) |  Pass  |  25.19%   |   104666.5   | 131030.5 |
|   http    |  100   |          40          |     1     | avg(requests_per_second) |  Fail  |  -27.99%  |   41068.5    | 29573.5  |
|   http    |  100   |          40          |    50     | avg(requests_per_second) |  Pass  |  -10.76%  |   160984.5   | 143668.0 |
|   http    |  100   |         200          |     0     | avg(requests_per_second) |  Pass  |  -9.86%   |   140269.0   | 126443.5 |
|   http    |  100   |         200          |     1     | avg(requests_per_second) |  Pass  |  30.15%   |   38852.0    | 50565.5  |
|   http    |  100   |         200          |    50     | avg(requests_per_second) |  Pass  |  -11.89%  |   147645.5   | 130094.5 |
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+----------+
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+---------+
| test_type | routes | conn_per_targetroute | keepalive |          metric          | result | deviation | OpenShiftSDN |   OVN   |
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+---------+
|   edge    |  100   |          1           |     0     | avg(requests_per_second) |  Pass  |  -14.61%  |   64454.5    | 55034.5 |
|   edge    |  100   |          1           |     1     | avg(requests_per_second) |  Fail  |  -59.37%  |   16390.0    | 6659.0  |
|   edge    |  100   |          1           |    50     | avg(requests_per_second) |  Fail  |  -24.41%  |   60963.0    | 46082.5 |
|   edge    |  100   |          40          |     0     | avg(requests_per_second) |  Pass  |  -11.03%  |   93206.5    | 82922.5 |
|   edge    |  100   |          40          |     1     | avg(requests_per_second) |  Fail  |  -60.53%  |   22190.5    | 8759.5  |
|   edge    |  100   |          40          |    50     | avg(requests_per_second) |  Fail  |  -20.08%  |   90001.5    | 71932.5 |
|   edge    |  100   |         200          |     0     | avg(requests_per_second) |  Pass  |  -3.41%   |   72936.0    | 70450.0 |
|   edge    |  100   |         200          |     1     | avg(requests_per_second) |  Fail  |  -56.28%  |   19330.5    | 8451.5  |
|   edge    |  100   |         200          |    50     | avg(requests_per_second) |  Pass  |  -11.42%  |   70538.0    | 62480.5 |
+-----------+--------+----------------------+-----------+--------------------------+--------+-----------+--------------+---------+
```

The CSV one looks like:

```csv
test_type,routes,conn_per_targetroute,keepalive,metric,result,deviation,OpenShiftSDN,OVN
http,100,1,0,avg(requests_per_second),Pass,-13.69%,59251.0,51141.5
http,100,1,1,avg(requests_per_second),Pass,-6.37%,21802.5,20414.5
http,100,1,50,avg(requests_per_second),Fail,-17.30%,56382.5,46628.5
http,100,40,0,avg(requests_per_second),Pass,25.19%,104666.5,131030.5
http,100,40,1,avg(requests_per_second),Fail,-27.99%,41068.5,29573.5
http,100,40,50,avg(requests_per_second),Pass,-10.76%,160984.5,143668.0
http,100,200,0,avg(requests_per_second),Pass,-9.86%,140269.0,126443.5
http,100,200,1,avg(requests_per_second),Pass,30.15%,38852.0,50565.5
http,100,200,50,avg(requests_per_second),Pass,-11.89%,147645.5,130094.5
test_type,routes,conn_per_targetroute,keepalive,metric,result,deviation,OpenShiftSDN,OVN
edge,100,1,0,avg(requests_per_second),Pass,-14.61%,64454.5,55034.5
edge,100,1,1,avg(requests_per_second),Fail,-59.37%,16390.0,6659.0
edge,100,1,50,avg(requests_per_second),Fail,-24.41%,60963.0,46082.5
edge,100,40,0,avg(requests_per_second),Pass,-11.03%,93206.5,82922.5
edge,100,40,1,avg(requests_per_second),Fail,-60.53%,22190.5,8759.5
edge,100,40,50,avg(requests_per_second),Fail,-20.08%,90001.5,71932.5
edge,100,200,0,avg(requests_per_second),Pass,-3.41%,72936.0,70450.0
edge,100,200,1,avg(requests_per_second),Fail,-56.28%,19330.5,8451.5
edge,100,200,50,avg(requests_per_second),Pass,-11.42%,70538.0,62480.5
test_type,routes,conn_per_targetroute,keepalive,metric,result,deviation,OpenShiftSDN,OVN
passthrough,100,1,0,avg(requests_per_second),Fail,-17.92%,72232.0,59288.0
passthrough,100,1,1,avg(requests_per_second),Fail,-41.12%,19871.0,11700.5
passthrough,100,1,50,avg(requests_per_second),Fail,-18.49%,70630.0,57567.0
passthrough,100,40,0,avg(requests_per_second),Fail,-18.76%,267765.0,217525.5
passthrough,100,40,1,avg(requests_per_second),Fail,-54.74%,45847.5,20750.5
passthrough,100,40,50,avg(requests_per_second),Fail,-24.36%,254821.5,192755.0
passthrough,100,200,0,avg(requests_per_second),Pass,-7.98%,205241.5,188861.5
passthrough,100,200,1,avg(requests_per_second),Fail,-25.92%,43207.5,32006.5
passthrough,100,200,50,avg(requests_per_second),Pass,-13.15%,206836.5,179646.0
test_type,routes,conn_per_targetroute,keepalive,metric,result,deviation,OpenShiftSDN,OVN
reencrypt,100,1,0,avg(requests_per_second),Pass,-13.76%,66611.0,57446.5
reencrypt,100,1,1,avg(requests_per_second),Fail,-63.95%,9899.0,3568.5
reencrypt,100,1,50,avg(requests_per_second),Fail,-24.22%,60534.0,45874.0
reencrypt,100,40,0,avg(requests_per_second),Pass,-10.91%,81379.5,72502.5
reencrypt,100,40,1,avg(requests_per_second),Fail,-41.93%,11297.0,6560.0
reencrypt,100,40,50,avg(requests_per_second),Fail,-17.30%,76874.5,63577.0
reencrypt,100,200,0,avg(requests_per_second),Pass,-4.12%,67594.0,64812.5
reencrypt,100,200,1,avg(requests_per_second),Fail,-40.58%,10181.0,6049.5
reencrypt,100,200,50,avg(requests_per_second),Pass,-12.88%,65881.5,57398.0
test_type,routes,conn_per_targetroute,keepalive,metric,result,deviation,OpenShiftSDN,OVN
mix,400,1,0,avg(requests_per_second),Pass,-9.62%,160425.5,144989.5
mix,400,1,1,avg(requests_per_second),Fail,-35.56%,27420.0,17670.0
mix,400,1,50,avg(requests_per_second),Fail,-18.44%,147568.0,120361.5
mix,400,20,0,avg(requests_per_second),Pass,-10.85%,148060.5,131990.0
mix,400,20,1,avg(requests_per_second),Fail,-42.48%,28687.5,16500.5
mix,400,20,50,avg(requests_per_second),Fail,-15.12%,146895.0,124678.0
mix,400,80,0,avg(requests_per_second),Pass,-7.42%,129215.0,119633.0
mix,400,80,1,avg(requests_per_second),Fail,-39.25%,27516.5,16716.5
mix,400,80,50,avg(requests_per_second),Fail,-23.76%,143691.0,109544.0

```

### CODE REVIEW

File: src/touchstone/compare.py [hunk 1]
```diff
@@ -141,8 +141,8 @@ def main(args):
     # Get indices from metadata map
     metadata_search_map = benchmark_instance.search_map_metadata
     metadata_dict = {}
-    if args.tolerancy_rules and len(args.uuid) < 2:
-        logger.critical("At least two uuids are required when tolerancy-rules flag is passed")
+    if args.tolerancy_rules and (len(args.uuid) < 2 or len(args.uuid) > 2):
+        logger.critical("Two uuids exactly are required when tolerancy-rules flag is passed")
```
Comment (jtaleric) [inline src/touchstone/compare.py:145]: This is a confusing log message. 

"No more than two UUIDs are allowed when passing the `tolerancy-rules` flag."

**Comment by mohit-sheth** [comment]
@Harshith-umesh can you also review this one?
in relation with https://github.com/cloud-bulldozer/e2e-benchmarking/pull/277

File: src/touchstone/compare.py (hunk 1 above)
Comment (Harshith-umesh) [reply src/touchstone/compare.py:145]: here we need exactly 2 uuid's , not more not less

**Review by Harshith-umesh — COMMENTED** [review]
LGTM

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Comment by mohit-sheth** [comment]
merging this, might need additional testing/PRs to get the gsheet generation working

**Review by Harshith-umesh — APPROVED** [review]
Tested this out , looks good. 
snappy link: http://ec2-54-191-102-253.us-west-2.compute.amazonaws.com:7070/index/harshith-umesh/testin.tar.gz
gsheet link: https://docs.google.com/spreadsheets/d/1i86DkMkk_dVK1QI3-_YOOEq0vQjPO6KomI5bPmRxwEs

**Pull request merged**